test/test_transforms.py

```diff
@@ -2381,10 +2381,7 @@ def test_transform_rb(self, include_forward, rbclass):
         )
         rb.extend(td)
 
-        if rbclass is ReplayBuffer:
-            storage = rb._storage._storage
-        else:
-            storage = rb._storage._storage.get("_data")
+        storage = rb._storage._storage[:]
```

Comment:
I would suggest to use copy() or clone() to be more explicit than [:] 

```suggestion
        storage = rb._storage._storage.copy()
```

Reply:
The `[:]` does not copy but it indexes all valid elements of the storage (ie the storage up to its maximum length).
Note also that this can return anything (eg a tensordict, a tensorclass or any pytree), but not a Storage object.